source/usage-examples/replaceOne.txt

```diff
@@ -3,3 +3,53 @@ Replace a Document
 ==================
 
 .. default-domain:: mongodb
+
+You can replace a document in a collection by using the ``ReplaceOne()``
+method. 
+
+The following example specifies a query filter and replacement document
+to the ``ReplaceOne()`` method, which matches a document in the
+``movies`` collection where the ``title`` field is "Shrek" and replaces
+it with a document that contains a ``title`` and ``plot`` field about
+the movie: 
```

Comment:
I think we should stick with haikus now. There are a lot of other fields in a `movie` document that we don't want to show, nor overwrite.